documentation/URL_SPEC.md

```diff
@@ -0,0 +1,159 @@
+| `rmp`    | Raw ramp values. If provided, overrides the `lut` field when calculating the ramp values. | Two float bin indices, formatted as `min:max` ([_see 'Binning'_](#binning))                    | `0:255`                                | `?c0=rmp:0:255`                     |
+| `lut`    | Lookup table (LUT) to map from volume intensity values to opacity.                        | [_See 'Lookup Table'_](#lookup-table-lut)                                                      | `0:255`                                | `?c0=lut:0:255`                     |
+
+### Binning
+
+When loaded, each channel's raw intensity values are sorted into one of 256
+bins, where the bin index `0` holds the minimum raw intensity value of the
+channel and the bin index `255` holds the maximum raw intensity value. Certain
+properties like `cps` (control points), `rmp` (ramp), and `lut` (lookup table)
+directly reference these bin indices.
+
+![Screenshot of Volume Viewer with a volume in advanced mode. A histogram with
+255 bins is visible, with a ramp increasing from 0 to 1 on the Y axis in the
+span of 53 to 136 on the X axis.](./assets/example_histogram.png)
+
+For example, the above control points are at bin indices `0`, `53`, `136`, and
+`255`. Depending on the range of the channel's values, these may
+represent different raw intensity values.
+
+| Min raw intensity | Max raw intensity | Bin `0` intensity | Bin `53` intensity | Bin `136` intensity | Bin `255` intensity |
+| ----------------- | ----------------- | ----------------- | ------------------ | ------------------- | ------------------- |
+| 0                 | 255               | 0                 | 53                 | 136                 | 255                 |
+| 50                | 200               | 50                | 103                | 166                 | 200                 |
+| 0                 | 1                 | 0                 | 0.207              | 0.533               | 1                   |
+
+### Lookup Table (`lut`)
+
+The `lut` property maps from volume intensity values to opacity, and is
+represented by a `min:max` pair. Values between the `min` and `max` ramp
+linearly from an opacity of `0` to `1`. The `min` and `max` can match any of the
+following:
+
+- Plain numbers are treated as bin indices, in a `[0, 255]` range.
+- `p{n}` represents a percentile, where `n` is a percentile in the `[0, 100]` range.
+- `m{n}` represents the volume's median intensity multiplied by `n / 100`.
+- `autoij` in either the min or max fields will use the ["auto" algorithm from
+  ImageJ](https://github.com/imagej/ImageJ/blob/7746fcb0f5744a7a7758244c5dcd2193459e6e0e/ij/plugin/frame/ContrastAdjuster.java#L816)
+  to select the min and max.
```

Comment:
Unsure if we can claim to _exactly_ implement ImageJ's algorithm, though @toloudis would know more